### PR TITLE
Harmonize README with FLOSS blocks

### PR DESCRIPTION
**Pull request by rodrigoalmeida94**
Branch from `change-name`

- [x] Update Makefile
- [x] Update Readme with instructions

Overall PR: https://github.com/up42/snap-polarimetric/pull/9
Tracked by https://geoinformationstore.atlassian.net/browse/UP-3132

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,36 +1,45 @@
...
+## Configuration for Makefile.
+SRC := blocks/kmeans_clustering
+UP42_DOCKERFILE := $(SRC)/Dockerfile
+UP42_MANIFEST := $(SRC)/UP42Manifest.json
+DOCKER_TAG := kmeans-clustering
+DOCKER_VERSION := latest
 
 VALIDATE_ENDPOINT := https://api.up42.com/validate-schema/block
 REGISTRY := registry.up42.com
-CURL := curl
```
Comment (perusio) [inline Makefile]: Why remove these variables? This is a good Makefile practice.

File: README.md [hunk 2]
```diff
@@ -36,59 +33,79 @@ and delivers a [GeoTIFF](https://en.wikipedia.org/wiki/GeoTIFF) file.
...
 ```bash
-make UID=$(pbpaste | cut -f 2 -d '/') USER=me@example.com DOCKER_TAG=land-cover-demo:mine-1.0
+make install
 ```
 
-In Linux do:
+### Run the tests
+
+This project uses [pytest](https://docs.pytest.org/en/latest/) for
+testing.  To run the tests, do as following:
```
Comment (perusio) [inline README.md]: Extra space after the `.`.

File: Makefile [hunk 3]
```diff
@@ -1,36 +1,45 @@
...
-DOCKER := docker
 
-all: validate build login push
+install:
```
Comment (perusio) [inline Makefile]: This now has implications for all the documentation around building a custom block,  https://docs.up42.com/going-further/custom-processing-block-dev.html. Please update that too. Thank you.

File: e2e.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+    # Print out bbox of one tile
+    GEOJSON_PATH = TEST_DIR / 'output' / 'data.json'
+
+    with open(str(GEOJSON_PATH)) as f:
+        FEATURE_COLLECTION = geojson.load(f)
+
+    print(FEATURE_COLLECTION.features[0].bbox)
+
+    OUTPUT_SNAP = TEST_DIR / 'output' / Path(
```
Comment (perusio) [inline e2e.py]: Isn't this a strange name for a variable. This block is totally different from SNAP AFAIK.

File: e2e.py [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+        FEATURE_COLLECTION.features[0].properties["custom.processing.land_cover"]
+    )
+
+    print(OUTPUT_SNAP)
```
Comment (perusio) [inline e2e.py]: Ditto.

File: e2e.py [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+    assert OUTPUT_SNAP.exists()
```
Comment (perusio) [inline e2e.py]: Re-ditto.

File: Makefile (hunk 1 above)
Comment (rodrigoalmeida94) [reply Makefile]: It is only used once and it won't change in the future, so not sure why we would make a variable out of it. The same thing for the docker commands. I think it makes the commands in the makefile more readable.

File: e2e.py (hunk 4 above)
Comment (rodrigoalmeida94) [reply e2e.py]: Good catch! Didn't change this.

File: Makefile (hunk 3 above)
Comment (rodrigoalmeida94) [reply Makefile]: You mean just the general change of name and functionality of the block? Not necessarily the change in the makefile correct?

**Pull request merged**